.github/scripts/bisection-config.sample.yaml

```diff
@@ -0,0 +1,15 @@
+# The sample bisection config that solves GH issue #51380
+
+# Start and end commits
+start: a87a1c1
+end: 0ead9d5
+# 10 percent regression
+threshold: 10
```

Comment:
what did you base this threshold on? is the 'noise level' about 5%?

Reply:
Unfortunately, although the overall score doesn't change 1% for the same build, it is common that individual tests result changes 5% or even more. For example, in this [issue](https://github.com/pytorch/pytorch/issues/51384) where regression only happens on BackgroundMatting model, other models' performance could change up to 9% (e.g., demucs and pytorch_struct). Therefore, I am only confident it is a real signal with >10% delta.